charts/kyverno/README.md

```diff
@@ -112,11 +115,11 @@ The following table lists the configurable parameters of the kyverno chart and t
+| `testImage.pullPolicy`             | image pull policy for test image (defaults to `image.pullPolicy`)                                                                                                                                                                                           | `nil`                                                                                                                                                                                                                                                                 |
+| `testImage.repository`             | repository for chart test image                                                                                                                                                                                                                          | `busybox`                                                                                                                                                                                                                                                                |
+| `testImage.tag`                    | tag for chart test image                                                                                                                                                                                                                                 | `nil`                                                                                                                                                                                                                                                                    |
 | `tolerations`                      | list of node taints to tolerate                                                                                                                                                                                                                          | `[]`                                                                                                                                                                                                                                                                     |
-| `securityContext`                  | security context configuration                                                                                                                                                                                                                           | `{}`                                                                                                                                                                                                                                                                     |
```

Comment:
`securityContext` is used in [deployment.yaml](https://github.com/kyverno/kyverno/blob/6d6dc8f1bcb6f53d2e80d1294d7c319f9c408471/charts/kyverno/templates/deployment.yaml#L31)
and `podSecurityStandard`, `podSecuritySeverity` and `podSecurityPolicies` are used in [value.yaml](https://github.com/kyverno/kyverno/blob/main/charts/kyverno/values.yaml)
So, why are these removed?

Reply:
My bad for not calling it out more explicitly: I moved those four items to different lines so the contents would all be in alphabetical order.